.vscode/launch.json

```diff
@@ -13,6 +13,9 @@
                 "test",
                 "${file}"
             ],
+            "skipFiles": [
+                "<node_internals>/**"
+            ],
```

Comment:
this helps with debugging. when you pause, debugger doesn't break on node internal code